internal/broker/config.go

```diff
@@ -27,11 +30,39 @@ const (
 	sshSuffixesKey = "ssh_allowed_suffixes"
 )
 
+func getDropInFiles(cfgPath string) ([]any, error) {
+	// Check if a .d directory exists and add all files in it to the sources.
+	dropInDir := cfgPath + ".d"
+	files, err := os.ReadDir(dropInDir)
+	if errors.Is(err, fs.ErrNotExist) {
+		return nil, nil
+	}
+	if err != nil {
+		return nil, err
+	}
+
+	var dropInFiles []any
```

Comment:
Why are using a `[]any` here? `ini.Load` should accept a slice of strings, no?

Reply:
no it does not. the function signature is

```
func Load(source interface{}, others ...interface{}) (*File, error)
```

we would have to convert each element in the string slice into an `interface{}`.